edit-post/store/reducer.js

```diff
@@ -44,7 +45,7 @@ export const preferences = combineReducers( {
 	panels( state = PREFERENCES_DEFAULTS.panels, action ) {
 		if ( action.type === 'TOGGLE_GENERAL_SIDEBAR_EDITOR_PANEL' ) {
 			return {
-				...state,
+				...( mapValues( state, () => false ) ),
```

Comment:
Very clever 👍 Maybe a bit [_too clever_](https://softwareengineering.stackexchange.com/questions/25276/why-is-cleverness-considered-harmful-in-programming-by-some-people). Could we include an inline comment explaining what's intended with this line?